plugin/src/main/kotlin/tanvd/grazi/utils/JLangUtils.kt

```diff
@@ -69,4 +80,35 @@ object LangToolInstrumentation {
             }
         }
     }
+
+    fun reloadEnglish() {
+        val reader = ClassReader(Resources.getStream("/org/languagetool/language/English.class"))
+        val writer = ClassWriter(ClassWriter.COMPUTE_MAXS)
+        reader.accept(EnglishChunkerInstrumentation(writer), 0)
+
+        val cls = writer.toByteArray()
+        with(ClassLoader::class.java.getDeclaredMethod("defineClass", String::class.java, ByteArray::class.java, Int::class.java, Int::class.java)) {
+            isAccessible = true
+            invoke(GraziPlugin.classLoader, "org.languagetool.language.English", cls, 0, cls.size)
+        }
+    }
+
+    private class EnglishChunkerInstrumentation(cv: ClassVisitor) : ClassVisitor(Opcodes.ASM5, cv) {
+        override fun visitMethod(access: Int, name: String?, descriptor: String?, signature: String?, exceptions: Array<out String>?): MethodVisitor {
+            val mv = cv.visitMethod(access, name, descriptor, signature, exceptions)
```

Comment:
`cv - classVisitor` and `mv - methodVisitor` -- let's be a little bit less c-plus-plus programmers)

Reply:
cv is protected field, but ok :)